scripts/index.js

```diff
@@ -167,6 +171,22 @@ function copyPublicFolder() {
   });
 }
 
+function runHook(hook) {
+  if (!hook || typeof hook !== 'string') {
+    return;
+  }
+
+  const exec = require('child_process').exec;
```

Comment:
I prefer to put the `require` at the top of the file, just to be consistent with what has already been done.